### ISSUE TITLE
Config file options don't seem to do anything

### ISSUE THREAD

**Issue by KipIngram** [kovidgoyal/kitty#7940]
I run Terminator for my main console sessions, but use kitty for some special purposes.  Changing font size, window size, window centering and so on in the config file and then starting kitty by typing kitty at my Terminator prompt doesn't seem to make any change.

I am running Fedora 40 under Wayland.

No other kitty is running when I do this.  Is this normal?  Is there more required to get new config options to "take"?

**Comment by kovidgoyal** [comment]
Nope if you all you need to do is edit kitty.conf.

**Comment by KipIngram** [comment]
That was what I found to be the case on my previous computer.  But I can't even seem to adjust the size of my window on this new one.

What I use kitty for is a set of "pop up tools" that I have set on hot keys.  Once, for example, brings up a set of frequently used "text snippets" and lets me select one for pasting - it uses a Linux tool called fzf.  I have it set up so it will just pop open a kitty window when I hotkey, and then when I hit enter on an item it copies it and closes the window.  I set the size and position of that window for aesthetic purposes.  That window doesn't seem to come up in the right place on the new computer, and I can't seem to adjust that or adjust the font size, which is slightly too large to suit me.

Anyway, thanks for your reply - I'll keep working on it.
